fsw/inc/to_lab_eventids.h

```diff
@@ -47,6 +47,8 @@
 #define TO_LAB_NOOP_INF_EID          18
 #define TO_LAB_TBL_ERR_EID           19
 #define TO_LAB_ENCODE_ERR_EID        20
+#define TO_LAB_SUBSCRIBE_CMD_ERR_EID 21
```

Comment:
What was the purpose of adding these two new event IDs?

Reply:
Just to have unique error EIDs instead of overloading a previous one.